src/main/resources/assets/music_control/lang/zh_cn.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Thanks to you, now almost all my mods are translated in Chinese! Thank you for this contribution too